google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -1378,6 +1387,8 @@ public static Builder newBuilder() {
   /**
    * Creates a new bucket.
    *
+   * Accepts an optional userProject {@link BucketTargetOption} option.
```

Comment:
Accepts an optional userProject {@link BucketTargetOption} option which defines the project id to assign operational costs.

Reply:
Done.